### PR TITLE
setup project

### PR DESCRIPTION
**Pull request by JDPablo**

### CODE REVIEW

File: .env [hunk 1]
```diff
@@ -0,0 +1 @@
+BOT_TOKEN=ODMyMzY3OTI3MzI3NDU3MzQy.YHixCg.SUSlB0aA5DLKTKI7zc3cgM--33s
```
Comment (ajones55555) [inline .env]: delete this file

File: .gitignore [hunk 2]
```diff
@@ -1,104 +1 @@
...
-
-# FuseBox cache
-.fusebox/
-
-# DynamoDB Local files
-.dynamodb/
-
-# TernJS port file
-.tern-port
+node_modules/
```
Comment (ajones55555) [inline .gitignore]: add `.env`

**Review by ajones55555 — CHANGES_REQUESTED** [review]

File: .env (hunk 1 above)
Comment (JDPablo) [reply .env]: deleted

File: .gitignore (hunk 2 above)
Comment (JDPablo) [reply .gitignore]: added

File: .env (hunk 1 above)
Comment (ajones55555) [reply .env]: my hero

**Review by ajones55555 — APPROVED** [review]

**Pull request merged**